docs/production-deployment/cloud/api-keys.mdx

```diff
@@ -196,6 +196,298 @@ tcld apikey disable --id <api-key-id>
 tcld apikey enable --id <api-key-id>
 ```
 
+### API Keys for Client Connections
+Temporal Cloud supports secure programmatic access through API Key Authentication.
+
+
+API Keys for Client Connections are a new capability that extend the existing API Keys feature to afford users the ability to auth(n/z) into Temporal Cloud's "data plane" from Client Connections (Temporal CLI, SDKs) using an API Key.
```

Comment:
i don't believe we've mentioned data plane in our public facing documentation before. just want to confirm we've cleared using this terminology with the relevant product folks.

Reply:
Passing by with citations:

./docs/production-deployment/cloud/audit-logging.mdx:Please note that Audit Logging only captures operations in the Temporal Cloud control plane and not the data plane.
./docs/production-deployment/self-hosted-guide/checklist.mdx:Since Temporal does not ship its own open source data plane, rolling your own can be complex and take effort to add.

Reply:
thanks -- i think we should still confirm with product first though :-)